src/main/java/seedu/address/experimental/model/util/BasicTemplates.java

```diff
@@ -1,23 +1,26 @@
 package seedu.address.experimental.model.util;
 
 import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
 
 import seedu.address.model.entity.Name;
+import seedu.address.model.entity.Stats;
 import seedu.address.model.entity.Template;
 
 /**
  * Basic templates to be used
  */
 public class BasicTemplates {
-    private static List<Template> templates = new ArrayList<>();
+    private static final List<Template> templates = new ArrayList<>();
 
     public static List<Template> getBasicTemplates() {
         templates.clear();
-        Template orc = new Template(new Name("orc"), 0.8f, 0.1f, 0.1f, new HashSet<>());
-        Template elf = new Template(new Name("elf"), 0.15f, 0.5f, 0.3f, new HashSet<>());
-        Template human = new Template(new Name("human"), 0.2f, 0.2f, 0.8f, new HashSet<>());
+        Stats orcStats = new Stats(15, 6, 1);
```

Comment:
Stats are hardcoded currently, do we want to have slightly randomized values based on these?